cipherscan

```diff
@@ -1056,6 +1094,22 @@ results=()
 # Call to the recursive loop that retrieves the cipher preferences
 get_cipher_pref $CIPHERSUITE
 
+# in case the server is intolerant to our big hello, try again with
+# a smaller one
+# do that either when the normal scan returns no ciphers or just SSLv2
+# ciphers (where it's likely that the limiting by OpenSSL worked)
+pref=(${cipherspref[0]})
+if [[ ${#cipherspref[@]} -eq 0 ]] || [[ ${pref[1]} == "SSLv2" ]]; then
```

Comment:
I'd like to add a warning that the site failed to negotiate the original Hello. It could be added at the end of the terminal output, but I'm not sure where it would fit in the json structure. Any thought?